src/assets/scss/utilities/_mui_overrides.scss

```diff
@@ -3,6 +3,16 @@
 }
 .MuiToggleButton-root {
   font-family: $body-font !important;
+  &.Mui-selected {
+    color: #FFF !important;
+    background-color: $blue !important;
+  }
+}
+.MuiButton-containedPrimary {
+  background-color: $blue !important;
+  a, a:hover, a:visited {
+    color: $white !important;
```

Comment:
There has got to be a better way to do this but I am not sure what it is. I could leave an apologetic comment but I don't think that would make it better